### ISSUE TITLE
Example from readme breaks links with ampersand, quote and other escaped characters

### ISSUE THREAD

**Issue by soon** [HubSpot/draft-convert#165]
The following example:

```
import React from "react";
import { Editor, EditorState, Modifier } from "draft-js";
import "./styles.css";
import { convertToHTML } from "draft-convert";

export default function App() {
  const [editorState, setEditorState] = React.useState(
    EditorState.createEmpty()
  );

  const htmlFromDocs = convertToHTML({
    entityToHTML: (entity, originalText) => {
      if (entity.type === "LINK") {
        return <a href={entity.data.url}>{originalText}</a>;
      }
      return originalText;
    }
  })(editorState.getCurrentContent());

  const validHtml = convertToHTML({
    entityToHTML: (entity, originalText) => {
      if (entity.type === "LINK") {
        return <a href={entity.data.url} />;
      }
      return originalText;
    }
  })(editorState.getCurrentContent());

  const handleAddLinkClick = () => {
    const contentState = editorState.getCurrentContent();
    const contentStateWithEntity = contentState.createEntity(
      "LINK",
      "MUTABLE",
      { url: "https://google.com/" }
    );
    const entityKey = contentStateWithEntity.getLastCreatedEntityKey();
    const contentStateWithLink = Modifier.applyEntity(
      contentStateWithEntity,
      editorState.getSelection(),
      entityKey
    );
    const newEditorState = EditorState.push(
      editorState,
      contentStateWithLink,
      "apply-entity"
    );
    setEditorState(newEditorState);
  };

  return (
    <div>
      <button onClick={handleAddLinkClick}>Add Link</button>
      <Editor editorState={editorState} onChange={setEditorState} />
      <p>{htmlFromDocs}</p>
      <p>{validHtml}</p>
    </div>
  );
}
```

Uses `return <a href={entity.data.url}>{originalText}</a>;` as [recommended in readme](https://github.com/HubSpot/draft-convert#converttohtml) to serialize anchors. However this leads to double escaping (https://codesandbox.io/s/beautiful-flower-yy4o9):

<img width="437" alt="Снимок экрана 2020-02-04 в 18 19 23" src="https://user-images.githubusercontent.com/1177331/73748301-ef9f5400-477a-11ea-8f61-2cb5004c96c2.png">

This can be fixed by omitting `originalText` and returning just `<a href={entity.data.url} />` (I suppose library will automatically walk over children and add them). 

TL;DR:
1. Seems like example in readme is broken
2. What is the correct way to serialize entities with nested content?
